### PR TITLE
update team leader onboarding to support current workshop setup

### PR DESCRIPTION
**Pull request by samumantha**
* replace hackmd with collaborative doc
* breakout room/session mentions are now less zoom specific
* replace mentions of "asking helpers to room" to "put question on collaborative document"

Zoom setup still mentioned in "background:teams" as "sometimes we also allow learners... ", other than that I did not quite find a reason to have a separate page for CR offered zoom room vs only self-organized teams. So no separate new page for now. 

The new screenshot is missing the annotations, but maybe they are not needed? If you think they are, can still add them

### CODE REVIEW

**Comment by samumantha** [comment]
Draft because: One page that is linked often, is hackmd-machanics , will do some updates there to call it collaborative doc, but leave the file name to not break other links for now.

File: hackmd-mechanics.md [hunk 1]
```diff
@@ -53,25 +53,25 @@ Other hints:
 - Please don't "select all", it highlights for everyone and adds a
```
Comment (MatiasJJ) [inline hackmd-mechanics.md:53]:
```suggestion
- NOTE: Please don't "select all", it highlights for everyone and adds a
```

File: hackmd-mechanics.md [hunk 2]
```diff
@@ -53,25 +53,25 @@ Other hints:
...
 ## Privacy
 
-- Assume the HackMD is **public and published: you never
+- Assume the collaborative document is **public and published: you never
   need to put your name there**.
 
-- The HackMD will be **published on the website afterwards**.  We will
+- The collaborative document will be **published on the website afterwards**.  We will
   remove all non-instructors names, but it's easier if you don't add
   it at the start.
```
Comment (MatiasJJ) [inline hackmd-mechanics.md:76]:
```suggestion
  it there in the first place.
```

File: team-leaders.md [hunk 3]
```diff
@@ -39,15 +39,13 @@ previous courses survey results:
...
 ## What will happen during workshop?
 
-Please review the workshop schedule, e.g.
-<https://coderefinery.github.io/2022-09-20-workshop/schedule/>. It
+Please review the workshop schedule on the main workshop page. It
 contains links to lesson material and exercises.
 
 **Interactive stream?**
 * Instructor Zoom is streamed and recorded (watch material, watch demo, type along)
 * Stream for everyone
```
Comment (MatiasJJ) [inline team-leaders.md:47]:
```suggestion
* Everyone follow stream during lessons
```

File: team-leaders.md [hunk 4]
```diff
@@ -39,15 +39,13 @@ previous courses survey results:
...
-* Learner Zoom for exercise sessions in breakout rooms (no recording)
-* Everybody interacts via HackMD
+* Everybody interacts via collaborative document
```
Comment (MatiasJJ) [inline team-leaders.md:48]:
```suggestion
* Exercises during the lessons are done individually or in teams
* Everybody interacts via collaborative document
```

File: team-leaders.md [hunk 5]
```diff
@@ -297,34 +273,29 @@ Most importantly, *you do not have to know everything* (we don't, either), but y
...
 ## Background: Teams
 
-We try to arrange people in teams which stay together for all breakout
-sessions on all days.  This allows people to form a bond and get the
-rooms started sooner.  We will try to keep you in the same breakout
-room as long as we can, but we give no promises and will rearrange as
-needed when people can't attend.
-
-We sometimes allow people to register as teams: This could for example be a
+We ask learners to register individually or as teams: This could for example be a
```
Comment (MatiasJJ) [inline team-leaders.md:276]:
```suggestion
A team could be for example a
```

File: team-leaders.md [hunk 6]
```diff
@@ -297,34 +273,29 @@ Most importantly, *you do not have to know everything* (we don't, either), but y
...
 group colleagues/friends where one of the team members has a bit of knowledge
```
Comment (MatiasJJ) [inline team-leaders.md:277]:
```suggestion
group of colleagues/friends where one of the team members has a bit of knowledge
```

File: team-leaders.md [hunk 7]
```diff
@@ -297,34 +273,29 @@ Most importantly, *you do not have to know everything* (we don't, either), but y
...
 on the tools presented in the workhop. This person can act as team leader
 for the workshop, but may still learn a thing or two themselves. In that way
 you can work with people you know and the barrier for asking questions and
 discuss together may be a bit lower than in a group of strangers. Since teams
```
Comment (MatiasJJ) [inline team-leaders.md:281]:
```suggestion
discuss together may be a bit lower than in a group of strangers. 
```

File: team-leaders.md [hunk 8]
```diff
@@ -297,34 +273,29 @@ Most importantly, *you do not have to know everything* (we don't, either), but y
...
 with team leaders are 'self-sustaining' we usually accept all team
```
Comment (MatiasJJ) [inline team-leaders.md:282]:
```suggestion

```

File: team-leaders.md [hunk 9]
```diff
@@ -297,34 +273,29 @@ Most importantly, *you do not have to know everything* (we don't, either), but y
...
-registrations, while registrations as single learner may be put on waiting list
-or, if applicable redirected to the stream, when they exceed our capacity based
-on the number of individual team leaders. 
+registrations.
```
Comment (MatiasJJ) [inline team-leaders.md:283]:
```suggestion
```

**Review by MatiasJJ — APPROVED** [review]
I wonder what happens to my suggestions if I approve but I'll approve anyway. I hope I can merge my suggestions still..

**Review by rkdarst — APPROVED** [review]

**Pull request merged**